templates/experiment.html

```diff
@@ -1,12 +1,50 @@
-<div class="card info-card">
+<!-- BEGIN INVALID AMP - don't try at home -->
+<div id="experiment-warning" class="card info-card">
   <h4>Experimental Mode</h4>
-  <p>The <code>{{component}}</code> component is currently in experimental mode and needs to be manually enabled:</p>
-
-  <ol>
-    <li>check the box under <a href="https://cdn.ampproject.org/experiments.html">CDN experiments</a>.
```

Comment:
Does AMP.toggleExperiment('AMP_CANARY') work or does it need to be called from cdn.ampproject.org?